app/src/main/java/org/fossasia/openevent/app/core/auth/login/LoginFragment.java

```diff
@@ -70,13 +69,24 @@ public void onStart() {
             if (!validator.validate())
                 return;
 
+            loginFragmentViewModel.getLogin().setEmail(binding.emailDropdown.getText().toString());
+            loginFragmentViewModel.getLogin().setPassword(binding.userPassword.getText().toString());
```

Comment:
This shouldn't be needed as you are already getting the email and password in login method

Reply:
This is needed because I have removed the setting of password and email ("@={login.email}" and "@={login.password}") from the XML file as the saved value in shared preference wasn't been shown in the edit text. So I am setting the values programmatically on click.

Reply:
I think if you let the previous method remain and bind the saved value correctly to View, it will work

Reply:
Still there and even two extra endpoints are added. We need to decrease calls in views, not increase